nixos/modules/services/continuous-integration/github-runner.nix

```
@@ -300,20 +299,35 @@ in
         RestrictSUIDSGID = true;
         UMask = "0066";
         ProtectProc = "invisible";
-        ProcSubset = "pid";
         SystemCallFilter = [
-          "~@debug"
-          "~@mount"
-          "~@privileged"
+          "~@capset"
```

Didn't catch that one in time but `capset` is also not a set but a system call:

```suggestion
          "~capset"
```